### ISSUE TITLE
Confusion in Synchronize file dialog

### ISSUE THREAD

**Issue by oscargus** [koppor/jabref#118]
In the synchronize file dialog, sometimes `Globals.FILE_FIELD.toUpperCase()` and sometimes `Localization.lang("file")` to denote what I think is the same thing. Probably a result of the previous ps and pdf fields. 

Examples of `Localization.lang("file")` (`fn`):
``` java
 Localization.lang("Autoset %0 links. Do not overwrite existing links.", fn), true);

 Localization.lang("This makes JabRef look up each %0 link and check if the file exists. If not, you will be given options<BR>to resolve the problem.", fn)
```

Example of 

``` java 
Localization.lang("Synchronize %0 links", Globals.FILE_FIELD.toUpperCase())
```

Shouldn't it be OK to hard code "file" in every position? (No need for separate translations.)

**Comment by simonharrer** [comment]
Hardcoding would be fine now.

**Comment by oscargus** [comment]
Fixed in JabRef/jabref#1020
